components/style/counter_style/mod.rs

```diff
@@ -149,7 +156,7 @@ impl<'a, 'b, 'i> AtRuleParser<'i> for CounterStyleRuleParser<'a, 'b> {
 
 macro_rules! checker {
     ($self:ident._($value:ident)) => {};
-    ($self:ident.$checker:ident($value:ident)) => {
+    ($self:ident. $checker:ident($value:ident)) => {
```

Comment:
This is touching a macro, and I'm not sure it's fine, actually... I guess it is but it looks fishy at best.